map/framework.cpp

```diff
@@ -2319,6 +2319,27 @@ bool Framework::ParseEditorDebugCommand(search::SearchParams const & params)
   return false;
 }
 
+namespace
+{
+vector<string> FilterNearbyStreets(vector<search::ReverseGeocoder::Street> const & streets)
```

Comment:
It is possible to do this in-place, in two or three places where it is needed. But we call it copy-paste and try to avoid it by using functions.